sc/social/like/plugins/facebook/browser.py

```diff
@@ -108,6 +104,46 @@ def typebutton(self):
             self.width = '55px'
```

Comment:
also, it is weird to calculate width into this property, I think this code could be extracted from here